ts/features/bonus/cgn/utils/filters.ts

```diff
@@ -0,0 +1,76 @@
+import { TranslationKeys } from "../../../../../locales/locales";
+
+export type CategoryType =
+  | "theater"
+  | "travel"
+  | "books"
+  | "health"
+  | "telco"
+  | "sport"
+  | "museum"
+  | "mobility";
+
+export type Category = {
+  type: CategoryType;
+  icon: string;
+  nameKey: TranslationKeys;
+};
+
+export const categories: ReadonlyArray<Category> = [
+  {
+    type: "theater",
+    icon: "io-theater",
+    nameKey: "bonus.cgn.merchantDetail.categories.theater"
+  },
+  {
+    type: "travel",
+    icon: "io-travel",
+    nameKey: "bonus.cgn.merchantDetail.categories.travel"
+  },
+  {
+    type: "mobility",
+    icon: "io-car",
+    nameKey: "bonus.cgn.merchantDetail.categories.mobility"
+  },
+  {
+    type: "telco",
+    icon: "io-phone-vibration",
+    nameKey: "bonus.cgn.merchantDetail.categories.telco"
+  },
+  {
+    type: "books",
+    icon: "io-books",
+    nameKey: "bonus.cgn.merchantDetail.categories.book"
+  },
+  {
+    type: "museum",
+    icon: "io-museums",
+    nameKey: "bonus.cgn.merchantDetail.categories.museum"
+  },
+  {
+    type: "sport",
+    icon: "io-sports",
+    nameKey: "bonus.cgn.merchantDetail.categories.sport"
+  },
+  {
+    type: "health",
+    icon: "io-wellness",
+    nameKey: "bonus.cgn.merchantDetail.categories.health"
+  }
+];
+
+export type OrderType = {
+  label: TranslationKeys;
+  value: string;
+};
+
+export const orders: ReadonlyArray<OrderType> = [
+  {
+    label: "bonus.cgn.merchantsList.filter.order.byDistance",
+    value: "distance"
+  },
+  {
+    label: "bonus.cgn.merchantsList.filter.order.byName",
+    value: "alphabetical"
+  }
+];
```

Comment:
```suggestion
export const orders : Record<string, OrderType> = {
  distance: {
    label: "bonus.cgn.merchantsList.filter.order.byDistance",
    value: "distance"
  },
  name: {
    label: "bonus.cgn.merchantsList.filter.order.byName",
    value: "alphabetical"
  }
};
```
What do you thing to model this data as an object instead of array
I suggest to avoid index access 
`orders[0].value`
that can be very dangerous if we'll change the array we can't get the error at compile time

If you need the array of the values:
```typescript
import _ from "lodash";
_.values(orders)
```